ruby/exercises/numeros_romanos.rb

```diff
@@ -0,0 +1,76 @@
+class Conversiones
+  @@respuesta = ""
+
+  def romanos(data)
+    return 'Numero invalido' unless data >= 1 && data <= 3999
+    @@respuesta = ""
+    @@respuesta = genera_busquedas(data)  
+  end
+
+  def numeros(data)
+    romanos = {'I': 1, 'V': 5, 'X': 10, 'L': 50, 'C': 100, 'D': 500, 'M': 1000}
+    numero = 0
+    valor_auxiliar = 0
+
+    data.chars.each do |chr|
+    return 'Literal invalida' unless romanos.keys.include?(chr.to_sym)
+
+      if valor_auxiliar != 0 && valor_auxiliar < romanos[chr.to_sym] 
+        numero -= valor_auxiliar
+        numero += (romanos[chr.to_sym] - valor_auxiliar)
+      else
+        numero += romanos[chr.to_sym]
+      end
+      valor_auxiliar = romanos[chr.to_sym]
+    end
+    numero
+  end
+
+  private 
+    def make_hash(opcion,literal, literal2, literal3)
+      hash = Hash.new 
```

Comment:
se prefiere el uso de "literales" para crear arreglos o hashes
```suggestion
      hash = {]
```